python-sdk/src/astro/sql/operators/append.py

```diff
@@ -63,6 +63,11 @@ def __init__(
 
     def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
         db = create_database(self.target_table.conn_id, table=self.source_table)
+        # currently, cross database operation is not supported
+        if (self.source_table and self.target_table) and (
```

Comment:
Would `self.source_table` or `self.target_table` be not set? If yes, do we need a separate error for them if they are unset?
e.g 
```
if not self.source_table or not self.source_table:
    raise some_error 
```

Reply:
In the case of a merge, an append operator source_table and target_table are required so the constructor should take care of it. Maybe if you want we can reduce it to.

```
if self.source_table.sql_type != self.target_table.sql_type:
    raise error
```